scripts/story-cleanup/story-cleanup.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python2
+
+import datetime
+import os
+import requests
+import sys
+import traceback
+
+from requests.exceptions import HTTPError
+
+PT_PROJID_PRI = os.getenv('PT_PROJID_PRIMARY')
+PT_PROJID_STR = os.getenv('PT_PROJID_STORAGE')
+PT_URLPFX_FMT = 'https://www.pivotaltracker.com/services/v5/projects/{}'
+PT_URLPFX_PRI = PT_URLPFX_FMT.format(PT_PROJID_PRI)
+PT_HDRS_BASE = {'X-TrackerToken':os.getenv('PT_API_TOKEN')}
+PT_PRMS_BASE = {'date_format':'millis', 'limit':50}
+PT_DAYS_AGED = 190 if 'PT_DAYS_AGED' not in os.environ else int(os.getenv('PT_DAYS_AGED'))
```

Comment:
Can you please rename these to make them a little more humanly readable.

eg. `PT_PROJID_STR` > `PT_PROJID_STORAGE`